src/docs/releases/2.0.0.md

```diff
@@ -246,6 +246,11 @@ Here are the updated signatures:
 
 These adjustments ensure compatibility and adherence to the latest conventions within the `SectionDisplayDriver` class.
 
+### GraphQL Module
+
+The GraphQL schema may change, because fields are now always added to the correct part. Previously, additional fields may have been added to the parent content item type directly. 
```

Comment:
> additional fields may have been added to the parent content item type directly

Not sure what it means, but having content type fields attached to the dynamic part (the implicit one matching the type name) was by design. Does it mean that an Article type will have to access an ArticlePart property in graphql to get the fields?

Reply:
This only affects fields that have been added to a statically typed part. E.g. if you add fields to the `HtmlBodyPart` they were added to the content type directly, which will also often failed because of duplicated field names. The dynamic part itself has not been changed and also collapsing fields to the parent content item type is left unchanged as well. So the schema will only change in rare cases, not commonly.
This is fixing bug #14592.